packages/react-devtools-shared/src/utils.js

```diff
@@ -489,12 +490,16 @@ export function getDisplayNameForReactElement(
       return 'StrictMode';
     case Suspense:
       return 'Suspense';
+    case SuspenseList:
+      return 'SuspenseList';
     default:
       const {type} = element;
       if (typeof type === 'string') {
         return type;
-      } else if (type != null) {
+      } else if (typeof type === 'function') {
         return getDisplayName(type, 'Anonymous');
+      } else if (type != null) {
+        return 'Anonymous';
```

Comment:
This should probably say something like `NotImplementedInDevTools` so that it's clear this needs to be fixed in React DevTools rather than in the user code.